src/components/atoms/SfCheckbox/SfCheckbox.html

```diff
@@ -0,0 +1,26 @@
+<div class="sf-checkbox">
+  <input
+    :id="id"
+    :name="name"
+    type="checkbox"
+    class="sf-checkbox__field"
+    :value="true"
+    v-model="isChecked"
+    :disabled="disabled" />
+  <!-- @slot Custom checkbox label markup (binds `checked` boolean) -->
```

Comment:
shouldn this be just "custom checkbox markup"? it's not only a label so the slot name is not accurate too. Probably this should be the default slot